src/common/plugin.ts

```diff
@@ -20,5 +20,5 @@ export abstract class Plugin implements IPlugin {
     return this.container.channelService.hasPermission(channelName, this.permission);
   }
 
-  public abstract async execute(message: IMessage): Promise<void>;
+  public abstract async execute(message: IMessage, args?: string[]): Promise<void>;
```

Comment:
What does this do?

Reply:
Nothing. just wanted to be more clear on the method for when we extend the class (never had all the correct parameters in this method for some reason (?))